components/_patterns/00-utilities/_variables.scss

```diff
@@ -15,21 +22,74 @@ $tangerine:          hsla(40, 100%, 44%, 1);
 $outrageous-orange:  hsla(13, 100%, 63%, 1);
 $persian-red:        hsla(0, 64%, 51%, 1);
 
-$body-color: $black;
-$link: $bermuda-grey;
-$link-hover: $dark-bermuda-grey;
+//
+// COLOR CONFIGURATION
+//
+
+  // COLOR PALETTE
+  // These are the colors you will use for design elements throughout the site.
+  // You can set the keys to whatever you like. For example, keys like "blue"
+  // will work.
+  //
+  // For each key, the only sub-key that is required is "color".
+  //
+  // Access via c() function. See _functions.scss
+  //
+  // Examples:
+  //    Access base color:
+  //      color: c(1);
+  //      color: c(blue);
+  //      background: linear-gradient(45deg, c(1), c(blue));
+  //
+  //    Accessing variants:
+  //      color: c(1, dark);
+  //      color: c(blue, light);
+  //      background: linear-gradient(45deg, c(1, light), c(blue, dark));
+  //
+  $c-palette: (
+    1: (
+      base: $bermuda-grey,
+      dark: $dark-bermuda-grey,
+      light: $light-bermuda-grey
+    ),
+    2: (
+      base: $emerald,
+      dark: darken($emerald, 10%),
+      light: lighten($emerald, 10%)
+    ),
+    persian-red: (
+      base: hsla(0, 64%, 51%, 1),
+      dark: hsla(0, 64%, 35%, 1),
+    ),
+    // Your turn!
+    // key: (
+    //   base: ,
+    //   dark: ,
+    //   light: ,
+    // ),
+  );
 
 //
-// UI ELEMENTS (form items, buttons etc)
+// MESSAGES
+// Set colors for Drupal messages.
+//
+$c-status: $emerald;
+$c-warning: $tangerine;
+$c-error: $persian-red;
+
 //
+// UI APPEARANCE
+// Set defaults for form items, buttons, etc.
+//
+$c-ui-lightest-grey: #F5F5F5;
```

Comment:
Anyone care if we remove the `ui` from the variable name? Just to make it smaller ...

Reply:
This section needs more consideration overall.